temporalio/worker/_worker.py

```diff
@@ -276,7 +283,9 @@ def __init__(
             # concurrent activities. We do this here instead of in
             # _ActivityWorker so the stack level is predictable.
             max_workers = getattr(activity_executor, "_max_workers", None)
-            if isinstance(max_workers, int) and max_workers < max_concurrent_activities:
+            if isinstance(max_workers, int) and max_workers < (
+                max_concurrent_activities or 0
+            ):
```

Comment:
I wonder if there is a way for us to introspect the tuner at this time to find its max activities